ci/docker/runtime_functions.sh

```diff
@@ -1922,6 +1922,20 @@ build_docs_beta() {
     popd
 }
 
+push_docs() {
+    version_name=$1
+    set -ex
+    pip3 install --user awscli
+    export PATH=~/.local/bin:$PATH
+    pushd docs/_build
+    wget https://mxnet-website-static-artifacts.s3.us-east-2.amazonaws.com/versions.zip && unzip versions.zip && rm versions.zip
+    mkdir $version_name && tar -xzf full_website.tgz -C $version_name --strip-components 1
+    mv $version_name versions
```

Comment:
check if version name folder already exists, if it does delete it

Reply:
Done